### PR TITLE
Add initial version for zwaad script readme/tutorial

### PR DESCRIPTION
**Pull request by matwey**

### CODE REVIEW

**Comment by matwey** [comment]
There is also [aad_readme_before_pf](https://github.com/snad-space/zwad/tree/aad_readme_before_pf) branch for previous (pre-coniferest) zwaad version.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,59 @@ zwadp -c iso --oid oid_m31.dat --oid fakes/oid_m31_fake.dat --feature feature_m3
...
+```
+
+```shell
+# Run the algorithm with M31 dataset
+zwaad --oid ../data/oid_m31.dat --feature ../data/feature_m31.dat --feature-names ../data/feature_m31.name --budget 4
+```
+
+The script output may be as the following:
+```
+Check https://ztf.snad.space/dr4/view/695211400124577 for details
```
Comment (hombit) [inline README.md:75]: Should we say that instead of this line browser can be open? It is what happened to me

File: README.md [hunk 2]
```diff
@@ -51,6 +51,59 @@ zwadp -c iso --oid oid_m31.dat --oid fakes/oid_m31_fake.dat --feature feature_m3
...
+695211400134512
+
+# Check answers log
+cat ./data/aad/answers_feature_m31.csv
+oid,is_anomaly
+695211400124577,1
+695211400034403,0
+695211400070144,0
+695211400134512,1
+```
```
Comment (hombit) [inline README.md:105]: After I run it I have the only new file `my_anomalies.txt`, no CSV file. And it is located in the current working directory, not in `data`

**Review by hombit — CHANGES_REQUESTED** [review]

**Comment by hombit** [comment]
Could you also check that you can reproduce this sequence of questions? In my case I have a different OIDs yielded.

**Review by hombit — APPROVED** [review]

**Pull request merged**